block/block.go

```diff
@@ -33,6 +36,42 @@ type Body struct {
 	Txs tx.Transactions
 }
 
+// GetMockBlock creates and returns a mock block for testing purposes.
```

Comment:
Please avoid adding test codes to the package source.

Reply:
I am not sure I understand the suggestion. Should I remove `GetMockBlock` entirely or just move it under tests?